docs/tutorial/using-checkbox/advanced-config.rst

```diff
@@ -0,0 +1,340 @@
+Advanced Config
+===============
+
+This section of the tutorial documents some useful configuration that are commonly
+found in the wild. This is by no means a complete list of all possible items that
+one can find in a configuration. Refer to the `launcher`_ section for more.
+
+Transport and Report
+====================
+
+Checkbox allows a configuration to specify how the test report should be represented.
+One usage of this feature is to output the final report to ``stdout`` in text form.
+Create a launcher like the following:
+
+.. code-block:: none
+
+  [test plan]
+  unit = com.canonical.certification::smoke
+  forced = yes
+
+  [test selection]
+  forced = yes
+
+  # note what comes after : is the name of the exporter
+  [exporter:text]
+  # this specific one exports text,
+  # you can also try com.canonical.plainbox::html
+  unit = com.canonical.plainbox::text
+
+  # similarly, what comes out : is the name of the transport
+  [transport:out_to_stdout]
+  # standard out, you can also try stderr for standard error
+  stream = stdout
+  type = stream
+
+  # also here, : delimits the name of the section
+  # this section tells Checkbox that we want a custom report
+  [report:screen]
+  # we used the exporter we defined before (by using its name)
+  exporter = text
+  # and we use the transport we defined as well
+  transport = out_to_stdout
+
+Launch checkbox, it should print a human readable output to the stream you chose!
+
+.. note::
+
+  Checkbox will ask you if you want to submit the ``screen`` report. This is
+  the Checkbox way of asking if you want it to produce that report. Respond yes.
+  See the example below to know how to avoid having to give confirmation
+  (using forced)
+
+
+It may be tempting to redirect this output to file manually, but it is possible to
+save it in a file using the same mechanism. Try the following launcher for
+instance to create a beautiful html report to a file:
+
+.. code-block:: none
+
+  [test plan]
+  unit = com.canonical.certification::smoke
+  forced = yes
+
+  [test selection]
+  forced = yes
+
+  [exporter:html]
+  unit = com.canonical.plainbox::html
+
+  [transport:out_to_file]
+  type = file
+  path = /tmp/output.html
+
+  [report:file_report]
+  exporter = html
+  transport = out_to_file
+  # This tells Checkbox to always produce this report
+  # without asking any confirmation
+  forced = yes
+
+Launch Checkbox, you should now have an html report to check out in
+``/tmp/output.html``.
+
+Of course you can have multiple exporters, as mentioned before, try the following
+launcher, it will produce one html report and two (equal) textual reports.
+
+.. code-block:: none
+
+  [test plan]
+  unit = com.canonical.certification::smoke
+  forced = yes
+
+  [test selection]
+  forced = yes
+
+  [exporter:text]
+  unit = com.canonical.plainbox::text
+
+  [transport:out_to_file1]
+  type = file
+  path = /tmp/upload.txt
+
+  [transport:out_to_file2]
+  type = file
+  path = ~/.last_result.txt
```

Comment:
Maybe add a comment to explain Checkbox can use stuff like `~` for current user directory? It might not be obvious what `~` represents (I'm not sure who our audience is on these tutorials...)

Reply:
well I think that the technical level is a bit higher than that, how can someone use checkbox if they do not know the basics of linux